groups/ecs/iam.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_iam_role" "ecs_execution_role" {
+  name = "ecs_execution_role"
+
+  assume_role_policy = <<EOF
```

Comment:
Have a look at [this](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role.html#example-of-using-data-source-for-assume-role-policy#example-of-using-data-source-for-assume-role-policy) and see if you can define the policy content using the data source instead. The output should be the same but the definition is a bit nicer to work with